posthog/models/prompt.py

```diff
@@ -4,14 +4,11 @@
 from django.utils import timezone
 
 
-class PromptSequenceState(models.Model):
```

Comment:
This rename leaves a pending model deletion migration. I think we need to add `UserPromptSequence` not rename `PromptSequence`

Reply:
I pushed a fix while you were typing this :D